bazel/api-golden/test_api_report.ts

```diff
@@ -51,17 +50,14 @@ const _origFetchAstModuleExportInfo = ExportAnalyzer.prototype.fetchAstModuleExp
  *   is scoped to a specific subpath/entry-point.
  */
 export async function testApiGolden(
-  goldenFilePath: string,
   indexFilePath: string,
-  approveGolden: boolean,
   stripExportPattern: RegExp,
   typePackageNames: string[] = [],
   packageJsonPath = resolveWorkspacePackageJsonPath(),
-  customPackageName?: string,
-): Promise<ExtractorResult> {
-  // If no `TEST_TMPDIR` is defined, then this script runs using `bazel run`. We use
-  // the runfile directory as temporary directory for API extractor.
-  const tempDir = process.env.TEST_TMPDIR ?? process.cwd();
+  customPackageName: string,
```

Comment:
If this argument is now required, does not that mean that `typePackageNames` and `packageJsonPath` both are not able to be used as default initializers?

Reply:
Fixed. good catch